ansible/roles/nginx-proxy/templates/openshift/nginx-configuration.yml

```diff
@@ -20,18 +20,23 @@ data:
 
     http {
 
-      log_format compression '$remote_addr - $remote_user [$time_local] '
-                           '"$request" $status $body_bytes_sent '
-                           '"$http_referer" "$http_user_agent" "$gzip_ratio"';
+      log_format logger-json escape=json '{"source": "nginx", "time": $msec, \
+					   "resp_body_size": $body_bytes_sent, "host": "$http_host", "address": "$remote_addr", \
+					   "request_length": $request_length, "method": "$request_method", "uri": "$request_uri", \
+             "status": $status,  "user_agent": "$http_user_agent", \
+             "resp_time": $request_time, "upstream_addr": "$upstream_addr"}';
 
       server {
 
           listen 8080;
 
-          # enable compression
```

Comment:
Is there any reason of removing a compression?

Reply:
I had misconception of the role of this setting. I replaced.